ui/pages/confirmations/components/confirm/info/shared/edit-gas-icon/edit-gas-icon.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { Dispatch, SetStateAction } from 'react';
+import {
+  Button,
+  ButtonSize,
+  ButtonVariant,
+  IconName,
+} from '../../../../../../../components/component-library';
+import { useTransactionModalContext } from '../../../../../../../contexts/transaction-modal';
+import { IconColor } from '../../../../../../../helpers/constants/design-system';
+import { useTransactionEventFragment } from '../../../../../hooks/useTransactionEventFragment';
+
+export const EditGasIcon = ({
```

Comment:
From the name alone, I assumed this was literally just a presentation icon.

Would it be more suggestive of the underlying logic and action if we called this `EditGasIconButton` for example?